test/markdown.js

```diff
@@ -356,5 +357,28 @@ describe('markdown processing', function () {
     it('processes a table', function () {
       assert.equal($('table').length, 1)
     })
+
+    it('properly renders long documents containing many consecutive HTML blocks', function (done) {
+      // the failure mode here is an infinite loop that mocha can't interfere
+      // with, so we need to set up an environment that can do it
+      //
+      function handleException (e) {
```

Comment:
aha! I better understand what the issue was now -- you couldn't even get timeouts?